lib/consumer.js

```diff
@@ -4,9 +4,13 @@ const EE = require('events').EventEmitter
 const inherits = require('util').inherits
 const Kafka = require('node-rdkafka')
 
-function Consumer (opts, log, next) {
+function Consumer (opts, log, next, topicConf) {
   this._log = log
-  this.consumer = new Kafka.KafkaConsumer(opts)
+  if (topicConf) {
+    this.consumer = new Kafka.KafkaConsumer(opts, topicConf)
+  } else {
+    this.consumer = new Kafka.KafkaConsumer(opts)
+  }
```

Comment:
Why this change? Could we just initialize topicConf to an empty object.

Reply:
done